bin/save_cache

```diff
@@ -27,12 +27,12 @@ fi
 S3_BUCKET_NAME=${CACHE_BUCKET_NAME-}
 if [ -z "$S3_BUCKET_NAME" ]; then
 	if [ -z "$BUILDKITE_PLUGIN_BASH_CACHE_BUCKET" ]; then
-		printenv | grep buildkite
-		echo "Reading bucket name from 'BUILDKITE_PLUGIN_BASH_CACHE_BUCKET'"
-		CACHE_BUCKET_NAME="$BUILDKITE_PLUGIN_BASH_CACHE_BUCKET"
-	else
 		echo "⛔Unable to save file to cache – no \$CACHE_BUCKET_NAME is set"
 		exit 1
+	else
+		echo "Reading bucket name from 'BUILDKITE_PLUGIN_BASH_CACHE_BUCKET'"
+		S3_BUCKET_NAME="$BUILDKITE_PLUGIN_BASH_CACHE_BUCKET"
+		echo "Using $S3_BUCKET_NAME as S3 Bucket"
```

Comment:
This should probably be outside the 2 `if`s, just like in `restore_cache`, to log this in all paths?

Reply:
Actually I see it's already on line 39… So just removing it from here should be enough 😉 
```suggestion
```